File: web/tests/reference/intervals/non-overlapping-intervals.py
def erase_overlap_intervals(intervals):
    intervals.sort(key=lambda iv: iv[1])
    removals = 0
    last_end = float("-inf")
    for start, end in intervals:
        if start >= last_end:
            last_end = end
        else:
            removals += 1
    return removals

File: web/tests/reference/intervals/non-overlapping-intervals.js
function eraseOverlapIntervals(intervals) {
  intervals.sort((a, b) => a[1] - b[1]);
  let removals = 0;
  let lastEnd = -Infinity;
  for (const [start, end] of intervals) {
    if (start >= lastEnd) lastEnd = end;
    else removals++;
  }
  return removals;
}
